inference_client/tasks/encode.py

```diff
@@ -19,11 +20,32 @@ class EncodeMixin:
     client: Client
 
     @overload
-    def encode(self, *, text: Union[str, Iterable[str]], **kwargs):
+    def encode(
+        self,
+        *,
+        text: Union[str, Iterable[str]],
+        on_done: Optional['CallbackFnType'] = None,
+        on_error: Optional['CallbackFnType'] = None,
+        on_always: Optional['CallbackFnType'] = None,
+        batch_size: Optional[int] = 8,
+        prefetch: Optional[int] = 100,
```

Comment:
what's the recommended value of `prefetch` for the encoding task?

Reply:
No absolute recommendation value for prefetch. But per the test performed in clip_client, prefetch=100 would cover most cases for extremely large request